### PR TITLE
Add abstract and bio for self

### PR DESCRIPTION
**Pull request by ibuclaw**
@acehreli - I've had my head down for a while.

### CODE REVIEW

File: 2017/talks/buclaw.dd [hunk 1]
```diff
@@ -29,14 +29,15 @@ VIDEO = $(VIDEO_NO)
 LEVEL = All
 
 ABSTRACT =
-$(P $(I Coming soon...))
+$(P Back in 2013, I did a number of talks in various conferences and meetings
+about GCC, GDB, and the D language support that I had been working on at the
```
Comment (joakim-noah) [inline 2017/talks/buclaw.dd]: gave a number of talks at

File: 2017/talks/buclaw.dd [hunk 2]
```diff
@@ -29,14 +29,15 @@ VIDEO = $(VIDEO_NO)
...
+time.  Four years has passed, so what has happened since then?  I can explain...
```
Comment (joakim-noah) [inline 2017/talks/buclaw.dd]: have passed

**Comment by joakim-noah** [comment]
Should probably get this in before the talk, @CyberShadow?

File: 2017/talks/buclaw.dd [hunk 3]
```diff
@@ -29,14 +29,16 @@ VIDEO = $(VIDEO_NO)
...
 BIO =
-Lead developer, project coordinator and the principal force behind the
-development of the GNU D Compiler from as far back as 2009, where have
-been responsible for the migration over from D1 to D2, as well as from
-GCC 3.x to GCC 4.x.  Others might be interested to know that outside
-hobby life I work on for a Cloud Services/Hosting company as a
-Technical Engineer, and unofficially use D on a frequent basis to
-develop interpreters, and various plug-ins to work within our
-infrastructure.
+$(Iain is an Infrastructure Administrator at Sociomantic Labs.  He has been
```
Comment (CyberShadow) [inline 2017/talks/buclaw.dd]: This doesn't look like valid DDoc :o

**Comment by CyberShadow** [comment]
@wilzbach Thx, I fixed the DDoc syntax and was waiting on Travis. BTW are you here this year?

**Comment by wilzbach** [comment]
> was waiting on Travis. 

Yep, actually doing the same - I guess it's overloaded as usual, but it looks go locally, so I am going ahead:

image


> BTW are you here this year? 

Yep, sitting directly next to the coffee :)

**Pull request merged**